Sources/Container+SwinjectStoryboard.swift

```diff
@@ -6,6 +6,9 @@
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
 
+#if os(iOS) || os(tvOS)
```

Comment:
```suggestion
#if canImport(UIKit)
```

This is a bit better IMHO.